bento2seldon4recsys/bento.py

```diff
@@ -143,14 +143,23 @@ def _send_feedback(
                 if 50 < request.jsonData.top_k and 50 <= len(relevance_scores):
                     ks.append(50)
 
+                extra = {}
+                if hasattr(self, "extra_monitor_request_info"):
+                    extra = {
+                        label: getattr(request.jsonData, label)
+                        for label in self.extra_monitor_request_info
+                    }
+
```

Comment:
This whole code is the same from the bento2seldon. We should instead, besides extracting the @property method like said in benton2seldon's PR, also include a method that will receive the request and return the dict. This way, such method would be invoked there when needed and also here.